tasks/01_install.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Install borgbackup
+  when: install_backup is not defined or install_backup
```

Comment:
As mentioned, disabling the whole role is best done when including it. Not for each task.